content/en/docs/collector/configuration.md

```diff
@@ -18,13 +18,13 @@ Familiarity with the following pages is assumed:
 The Collector consists of four components that access telemetry data:
 
 - [Receivers](#receivers)
-  <img width="32" class="img-initial" src="/img/logos/32x32/Receivers.svg">
+  <img width="32" alt="" class="img-initial" src="/img/logos/32x32/Receivers.svg">
```

Comment:
Following https://www.w3.org/WAI/tutorials/images/decision-tree/ images that are purely decorative can have an empty alt attribute